cmd/lifecycle/creator.go

```diff
@@ -167,13 +167,10 @@ func (c *createCmd) Exec() error {
 	)
 	if c.platform.API().AtLeast("0.7") {
 		cmd.DefaultLogger.Phase("ANALYZING")
-		analyzedMD, err = analyzeArgs{
-			additionalTags:   c.additionalTags,
```

Comment:
I guess we can get away without `additinalTags` and `outputImageRef` because creator does validation on its own?

Reply:
Though it could be worth consolidating the validation into an additional package or service that could be unit tested. That could help further our goal of moving things out of the `cmd` package.